sparta/test/PriorityQueue/PriorityQueue_test.cpp

```diff
@@ -0,0 +1,210 @@
+
+#include "sparta/resources/PriorityQueue.hpp"
+
+#include <cinttypes>
+#include <iostream>
+#include <chrono>
+
+#include "sparta/utils/SpartaTester.hpp"
+
+constexpr bool TESTPERF = false;
+
+void test_defafult_pq()
+{
+    sparta::PriorityQueue<uint32_t> pqueue;
+
+    for(auto i : {1,3,2,5,6,4,8,7}) {
+        pqueue.insert(i);
+    }
+
+    EXPECT_EQUAL(pqueue.size(), 8);
+
+    EXPECT_EQUAL(pqueue.top(), 1);
+    pqueue.pop();
```

Comment:
Minor but to be consistent with the other lines of code:

```
pqueue.pop(); // 1
```

Reply:
added